scripts/install_foundry.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+export XDG_CONFIG_HOME=$HOME
+
+FOUNDRY_VERSION=v0.1.0
+curl -L https://raw.githubusercontent.com/ava-labs/foundry/${FOUNDRY_VERSION}/foundryup/install > /tmp/foundry-install-script
+sed -i "s/\/ava-labs\/foundry\/master\/foundryup/\/ava-labs\/foundry\/${FOUNDRY_VERSION}\/foundryup/g" /tmp/foundry-install-script
+cat /tmp/foundry-install-script | bash
+echo "export PATH=\"$PATH:$HOME/.foundry/bin\"" >> ~/.bashrc
```

Comment:
why do we need to echo into bash here, with no source following it.

Reply:
Line 13? `bashrc` is a file.

Reply:
yup why do we need to echo into `.bashrc` here? We don't have a following `source bashrc` so don't think this would take into effect, and we also export it specifically in the next line.

Reply:
Ohhh sorry I misunderstood. The idea is that the next time a new shell is opened, it will be able to find it. Though I think the foundry install script may do this on its own, I'll try removing it...

Reply:
Seems to be fine without it, so it's gone.